collector/processor/coldstartprocessor/go.mod

```diff
@@ -1,17 +1,20 @@
 module github.com/open-telemetry/opentelemetry-lambda/collector/processor/coldstartprocessor
 
-go 1.23
+go 1.23.0
+
+toolchain go1.23.6
```

Comment:
I don't think we want these changes...

Reply:
Yeah, I think, once this is addressed, we can merge this PR.

Reply:
Ah, must have been the language server updating this. Fixed in the latest revision